apps/site/downloadUtils/index.tsx

```diff
@@ -152,6 +137,8 @@ export const OPERATING_SYSTEMS: Array<DownloadDropdownItem<UserOS>> = [
   },
 ];
 
+// Here the list of Install Method Dropdown items are defined !
+
 export const INSTALL_METHODS: Array<
```

Comment:
Let's move this into the JSON

Reply:
You are talking about the commemts ot the INSTALL_METHODS logic ?